Sources/Build/Build.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import Subprocess
+
+@main
+struct Build {
+    static func main() async {
+        do {
+            try await Self.withVersion(in: "Sources/VaporToolbox/Version.swift", as: Self.currentVersion) {
+                _ = try await Subprocess.run(
+                    .path("/usr/bin/env"),
+                    arguments: [
+                        "swift", "build",
+                        "--disable-sandbox",
+                        "--configuration", "release",
+                        "-Xswiftc", "-cross-module-optimization",
+                    ]
+                )
+            }
+        } catch {
+            print("Error: \(error)")
+            exit(1)
+        }
+    }
+
+    static func withVersion(in file: String, as version: String, _ operation: @Sendable () async throws -> Void) async throws {
+        let fileURL = URL(filePath: file)
+        let originalFileContents = try String(contentsOf: fileURL, encoding: .utf8)
+
+        try originalFileContents
+            .replacingOccurrences(of: "nil", with: "\"\(version)\"")
+            .write(to: fileURL, atomically: true, encoding: .utf8)
+
+        do {
+            try await operation()
+        } catch {
+            // Revert the file change
+            try? originalFileContents.write(to: fileURL, atomically: true, encoding: .utf8)
+            throw error
+        }
+    }
+
+    static var currentVersion: String {
+        get async throws {
+            let tagResult = try await Subprocess.run(.path("/usr/bin/env"), arguments: ["git", "describe", "--tags", "--exact-match"])
```

Comment:
same here with `env`.
```suggestion
let tagResult = try await Subprocess.run(.name("git"), arguments: ["describe", "--tags", "--exact-match"])
```